src/allencell_ml_segmenter/thresholding/thresholding_view.py

```diff
@@ -0,0 +1,349 @@
+            self._autothreshold_radio_button.isChecked()
+        )
+        self._enable_specific_threshold_widgets(
+            self._specific_value_radio_button.isChecked()
+        )
+
+        self._apply_save_button.setEnabled(
+            self._specific_value_radio_button.isChecked()
+            or self._autothreshold_radio_button.isChecked()
+        )
+
+    def _disable_all_thresholding(self) -> None:
+        # TODO handle disabling ui for thresh autothresh
+        self._thresholding_model.disable_all()
+
+    def _check_able_to_threshold(self) -> bool:
+        able_to_threshold: bool = True
+        # Check to see if output directory is selected
+        if self._thresholding_model.get_output_directory() is None:
+            show_info("Please select an output directory first.")
+            able_to_threshold = False
+
+        # Check to see if input images / directory of images are selected
+        if self._thresholding_model.get_input_mode() is None:
+            show_info("Please select an input mode first.")
+            able_to_threshold = False
+        else:
+            if (
+                self._thresholding_model.get_input_mode()
+                == InputMode.FROM_NAPARI_LAYERS
+                and len(self._thresholding_model.get_selected_idx()) == 0
+            ):
+                show_info("Please select on screen images to threshold.")
+                able_to_threshold = False
+            elif (
+                self._thresholding_model.get_input_mode()
+                == InputMode.FROM_PATH
+                and self._thresholding_model.get_input_image_path() is None
+            ):
+                show_info("Please select a directory to threshold.")
+                able_to_threshold = False
+
+        # check to see if thresholding method is selected
+        if (
+            not self._thresholding_model.is_threshold_enabled()
+            and not self._thresholding_model.is_autothresholding_enabled()
+        ):
+            show_info("Please select a thresholding method first.")
+            able_to_threshold = False
+
+        return able_to_threshold
+
+    def _save_thresholded_images(self) -> None:
+        output_dir: Optional[Path] = (
+            self._thresholding_model.get_output_directory()
+        )
+        selected_images_viewer: list[Path] = []
+        for idx, layer in enumerate(
+            self._viewer.get_all_layers_containing_prob_map()
+        ):
+            if idx in self._thresholding_model.get_selected_idx():
+                selected_images_viewer.append(layer.metadata["source_path"])
+        self._thresholding_model.set_selected_paths(selected_images_viewer)
+
+        if output_dir is not None and self._check_able_to_threshold():
+            # progress tracker is tracking number of images saved to the thresholding folder
+            progress_tracker: PredictionFolderProgressTracker = (
+                PredictionFolderProgressTracker(
```

Comment:
`PredictionFolderProgressTracker` seems like this shoudl have a more general name now that we also use it in thresholding.